### ISSUE TITLE
issue_command args parser does not support native str als value

### ISSUE THREAD

**Issue by AnsgarSchmidt** [yamcs/python-yamcs-client#30]
Hi,

I ran into the following problem while I tried to send a str as part of a TC Command Argument.

```
client = YamcsClient('http://localhost:8090/')
processor = client.get_processor(instance='test', processor='realtime')
processor.issue_command("/a/b/c", args = {"key": "value"})
```

Error Message:

```
Traceback (most recent call last):
  File "venv/lib/python3.12/site-packages/yamcs/client/core/helpers.py", line 201, in to_argument_value
    obj = [to_argument_value(x, force_string=True) for x in value]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  [Previous line repeated 992 more times]
RecursionError: maximum recursion depth exceeded
```

So the problem is the helpers function does interpret the "Value" as a collections.abc.Sequence
and runs into an endless loop.

I will create a pull request with a fix for it.

Greetings
Ansgar

**Comment by fqqb** [comment]
Thanks, good find. I wll do a quick release for this one.

**Comment by fqqb** [comment]
Merged, and released in v1.11.1
